Ports/nesalizer/package.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=nesalizer
-version=git
-workdir=${port}-master
-makeopts=("CONF=release" "EXTRA=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2")
-files="https://github.com/SerenityPorts/nesalizer/archive/master.zip nesalizer-master.zip"
+version=5bb045845a5cc829a78b7384f848fdd886cd98c8
+files="https://github.com/ulfalizer/nesalizer/archive/${version}.tar.gz ${port}-${version}.tar.gz 4282cb0e4af0585af4a594dfa30b2e350dd0efc39e5bc2d8312f637f50397107"
+auth_type=sha256
+
 depends=("SDL2")
+makeopts+=("CONF=release" "CFLAGS=--sysroot=${SERENITY_INSTALL_ROOT}" "EXTRA=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2")
```

Comment:
This should probably stay as-is. We are mainly concerned with restoring the patches here, and `--sysroot` should already be handled by the default `${CC}` value anyways if I recall correctly.

Reply:
I omitted the SDL location completely and it still runs fine. Is that OK?